commercetools-models/src/main/java/io/sphere/sdk/carts/queries/ShippingRateInputQueryModel.java

```diff
@@ -0,0 +1,10 @@
+package io.sphere.sdk.carts.queries;
+
+import io.sphere.sdk.queries.OptionalQueryModel;
+import io.sphere.sdk.queries.QueryModel;
+import io.sphere.sdk.queries.StringQuerySortingModel;
+
+public interface ShippingRateInputQueryModel<T> extends OptionalQueryModel<T> ,QueryModel<T> {
```

Comment:
Have you tried to use a generated query model?

Reply:
Actually this should be removed 👍

Reply:
Actually for optional attribute the query model has to be created manually, maybe its a limitation of the generator, actually i didnt found out how to informit that the attribute is optional 😢

Reply:
I see 😢 But then I'm fine with your code 👍